diff --git a/drivers/pci/quirks.c b/drivers/pci/quirks.c
index e2a879e93d86..62b754253318 100644
--- a/drivers/pci/quirks.c
+++ b/drivers/pci/quirks.c
@@ -5118,3 +5118,7 @@ SWITCHTEC_QUIRK(0x8573);  /* PFXI 48XG3 */
 SWITCHTEC_QUIRK(0x8574);  /* PFXI 64XG3 */
 SWITCHTEC_QUIRK(0x8575);  /* PFXI 80XG3 */
 SWITCHTEC_QUIRK(0x8576);  /* PFXI 96XG3 */
+
+DECLARE_PCI_FIXUP_HEADER(0x10de, 0x1401, quirk_no_bus_reset);
+DECLARE_PCI_FIXUP_HEADER(0x10de, 0x0fba, quirk_no_bus_reset);
+DECLARE_PCI_FIXUP_HEADER(0x10ec, 0x5760, quirk_no_bus_reset);
