diff --git a/drivers/pci/quirks.c b/drivers/pci/quirks.c
index cc616a5f6a8f..eb4ecd075a02 100644
--- a/drivers/pci/quirks.c
+++ b/drivers/pci/quirks.c
@@ -5199,3 +5199,7 @@ static void quirk_reset_lenovo_thinkpad_p50_nvgpu(struct pci_dev *pdev)
 DECLARE_PCI_FIXUP_CLASS_FINAL(PCI_VENDOR_ID_NVIDIA, 0x13b1,
 			      PCI_CLASS_DISPLAY_VGA, 8,
 			      quirk_reset_lenovo_thinkpad_p50_nvgpu);
+
+DECLARE_PCI_FIXUP_HEADER(0x10de, 0x1401, quirk_no_bus_reset);
+DECLARE_PCI_FIXUP_HEADER(0x10de, 0x0fba, quirk_no_bus_reset);
+DECLARE_PCI_FIXUP_HEADER(0x10ec, 0x5760, quirk_no_bus_reset);
